airflow/providers/snowflake/hooks/snowflake.py

```diff
@@ -321,6 +320,21 @@ def set_autocommit(self, conn, autocommit: Any) -> None:
     def get_autocommit(self, conn):
         return getattr(conn, "autocommit_mode", False)
 
+    @staticmethod
+    def split_sql_string(sql: str) -> list[str]:
```

Comment:
Why this function was added?

Reply:
For most hooks, we split queries into statements using `sqlparse` but Snowflake has its own splitter.